sentinelhub/migration_from_scihub_guide.ipynb

```diff
@@ -310,51 +319,62 @@
    "id": "a1c7e1bd-9790-46b7-b5f6-122a07117071",
    "metadata": {},
    "source": [
-    "### Retrieving your Keycloak Token"
+    "### Retrieving your Access Token"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "17b2e052-6962-4881-8fd0-de9ad3dc5771",
    "metadata": {},
    "source": [
-    "Whilst no credentials are needed to search the catalogue; in order to download products from Copernicus Data Space Ecosystem catalogue using OData and OpenSearch API users are required to have a Keycloak token. This token can be generated in both Linux and Window OS using either `cURL` or `python` script. You can generate this token using the following code block (more information can be found in our [documentation](https://documentation.dataspace.copernicus.eu/APIs/Token.html)).\n",
+    "Whilst no credentials are needed to search the catalogue; in order to download products from Copernicus Data Space Ecosystem catalogue using OData and OpenSearch API users are required to have a Access token. This token can be generated in both Linux and Window OS using either `cURL` or `python` script. You can generate this token using the following code block (more information can be found in our [documentation](https://documentation.dataspace.copernicus.eu/APIs/Token.html)).\n",
```

Comment:
Not a native speaker, but I'd put "an Access token" here.

Reply:
updated "a" to "an"